src/app/models/imports.go

```diff
@@ -0,0 +1,119 @@
+package models
+
+import (
+	"context"
+	"shin/src/database"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+	"github.com/lib/pq"
+)
+
+type Import struct {
+	ID         uuid.UUID    `db:"id" json:"id"`
+	Target     ImportTarget `db:"target" json:"target"`
+	UserID     uuid.UUID    `db:"user_id" json:"user_id"`
+	Entities   []uuid.UUID  `db:"-" json:"entities"`
+	TotalCount int          `db:"total_count" json:"total_count"`
+	Count      int          `db:"count" json:"count"`
+	Status     ImportStatus `db:"status" json:"status"`
+	CreatedAt  time.Time    `db:"created_at" json:"created_at"`
+
+	EntitiesArray []uint8 `db:"entities" json:"-"`
+}
+
+func (Import) TableName() string {
+	return "imports"
+}
+
+func (Import) FetchQuery() string {
+	return "imports/fetch"
+}
+
+func (i *Import) Scan(rows *sqlx.Rows) error {
+	return rows.StructScan(i)
+}
+
+func (i *Import) Create(ctx context.Context) error {
+
+	rows, err := database.Query(
+		ctx,
+		"imports/create",
+		i.UserID, i.Target, i.TotalCount,
+	)
+
+	if err != nil {
+		return err
+	}
+
+	defer rows.Close()
+	for rows.Next() {
+		if err := i.Scan(rows); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (i *Import) Update(ctx context.Context) error {
+
+	rows, err := database.Query(
+		ctx,
+		"imports/update",
+		i.ID, pq.Array(i.Entities),
+	)
+
+	if err != nil {
+		return err
+	}
+
+	defer rows.Close()
+	for rows.Next() {
+		if err := i.Scan(rows); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (i *Import) Append(ctx context.Context, entity uuid.UUID) error {
+
+	tx, err := database.GetDB().Beginx()
+	if err != nil {
+		return err
+	}
+	rows, err := database.TxQuery(ctx,
```

Comment:
you don't need transactional query just simple query `database.Query` would do the job transactions just use to do couple of  trx on database